scripts/validate_docstrings.py

```diff
@@ -239,6 +240,11 @@ def get_api_items(api_doc_fd):
 
 
 class Docstring:
+    DIRECTIVES = ["deprecated", "versionadded", "versionchanged"]
+    DIRECTIVE_WITHOUT_TWO_COLONS = re.compile(
```

Comment:
We define similar constants at the top of the file, I'd keep these two there.

I find the name `DIRECTIVE_WITHOUT_TWO_COLONS` a bit misleading. If I understand correctly it's the same exact directives than the previous, but as compile expression to detect them. I understand your reasoning for the name, and it may be the best option, but reading the two constants gives me the impression that it's a different set of directives.

Reply:
Agreed.  We should rename this.

Reply:
Is it OK as `DIRECTIVE_PATTERN`?

Reply:
sounds good to me